server/db/models/Gig.js

```diff
@@ -10,6 +10,49 @@ module.exports = function (sequelize, DataTypes) {
   }, {
     classMethods: {
 
+      getGigs: function (organizationId) {
+        return Gig.findAll({
+          where: {
+            organizationId: organizationId
+          }
+        });
+      },
+
+      createGig: function (gigParams) {
+        return Gig.create(gigParams);
+      },
+
+      getGigInfo: function (id) {
+        return Gig.find({
+          where: {
+            id: id
+          }
+        });
+      },
+
+      updateGigInfo: function (id, params) {
+        return Gig.update(params, {
+          where: {
+            id: id
+          }
+        });
+      },
+
+      deleteGig: function (id) {
+        return Gigs.find({
+          where: {
+            id: id
+          }
+        }).then(function (gig) {
+          if ( gig === null ) {
+            return false;
+          } else {
+            gig.destroy().then(function () {
+              return true;
```

Comment:
Good catch I forgot to return the promise on gig.destroy(). Will update the PR with this fix shortly